src/components/MovieCard.vue

```diff
@@ -1,23 +1,76 @@
 <template>
-  <div class="crop">
+  <div class="my-card">
     <img :src="require(`@/assets/${imgSrc}`)" alt="" @click="routerFunction()">
-	</div>
+    <div class="my-content">
+      <button><i class="fas fa-play"></i></button>
+      <button><i class="fas fa-check"></i></button>
+      <button><i class="far fa-thumbs-up"></i></button>
+      <button><i class="far fa-thumbs-down"></i></button>
+      <p>2h35m</p>
+      <p>Drama, Action</p>
+    </div>
+  </div>
 </template>
 
 <script>
-export default ({
+  export default ({
     props: ['id', 'imgSrc'],
     methods: {
-        routerFunction() {
-            let route = this.$router.resolve({ path: "/details/"+this.id});
-            window.open(route.href);
-        }
+      routerFunction() {
+        let route = this.$router.resolve({
+          path: "/details/" + this.id
+        });
+        window.open(route.href);
+      }
     }
-})
+  })
+
 </script>
 
 <style scoped>
-  .crop:hover {
+  .my-card {
+    position: relative;
+    margin-right: 30px;
+  }
+  .my-card:hover {
     cursor: pointer;
   }
+  img {
+    transition: transform .3s;
+  }
+  img:hover {
+    transition: transform .3s;
+    -ms-transform: scale(1.2);
+    -webkit-transform: scale(1.2);
+    transform: scale(1.2);
+  }
+  img:hover + .my-content {
+    z-index: 10;
+  }
+  .my-content {
+    background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.6));
+    color: white;
+    width: 100%;
+    position: absolute;
```

Comment:
why not relative?

Reply:
i'm setting the position relative to its parent so the parent has position: relative and this has position:absolute